SecondStage/src/main/java/errorandexceptions/Faculty.java

```diff
@@ -0,0 +1,64 @@
+package errorandexceptions;
+
+import errorandexceptions.exceptions.FacultyHasNoGroupsException;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Random;
+
+public class Faculty {
+    private String name;
+    private List<Group> groups;
+    private static Random random = new Random();
```

Comment:
According to Code Conventions for the Java Programming Language, the parts of a class or interface declaration should appear in the following order:

Class (static) variables. First the public class variables, then protected, then package level (no access modifier), and then private.